LiskHQ/lisk-sdk
# Remove signature and signSignature, and multi-signature handling from Base Transaction, and replace with signatures verification - Closes #4838

### What was the problem?

This PR resolves #4838 

### How was it solved?
- Removed usage of `signature` and `signSignature`
- Replaces `signature` logic with `signatures`
- Update signature verification
- Updated all the test to use `signatures`
- Removed unwanted `multi-signature` code
### How was it tested?

- Build should be green
- Send type 8 to 12 and check if they get accepted
